.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
http://libpng.sourceforge.net links to https://github.com/glennrp/libpng, which has a `libpng16` branch. the current `HEAD` of that branch is probably a good target for the submodule?